homeassistant/components/switchbot/sensor.py

```diff
@@ -93,4 +109,9 @@ def __init__(
     @property
     def native_value(self) -> str:
         """Return the state of the sensor."""
+        if self._sensor == "temp" and self.data["data"].get("temp"):
+            if self.data["data"].get("fahrenheit"):
+                return self.data["data"]["temp"].get("f")
+            return self.data["data"]["temp"].get("c")
```

Comment:
It's prettier if you create a variable:
```suggestion
        if self._sensor == "temp" and (temp := self.data["data"]["temp"]):
            if self.data["data"].get("fahrenheit"):
                return temp.get("f")
            return temp.get("c")
```